Makefile

```diff
@@ -6,8 +6,9 @@ check:
 check-pypi-packaging:
 	podman run --rm -ti -v $(CURDIR):/src -w /src registry.fedoraproject.org/fedora:29 bash -c '\
 		pip3 install . \
-		&& ab --help \
-		&& ab build --help'
+		&& ansible-bender --help \
+		&& ansible-bender build --help' \
+		&& pip3 freeze | grep ansible-bender
```

Comment:
Not sure where to add this but here's the metadata validation command `twine check .tox/dist/*`

Reply:
I actually don't use tox since python3 only, I'll add it to todo

Reply:
why? how is this related to py3?

Reply:
I meant your example where `.tox` was in it. Just today I found out what does the command actually do.